src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,9 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    /**
+     * Returns true if the data is mutated
```

Comment:
You have added the header comments correctly. But it can be more accurate. Think of a potential case where you will add a duplicate person to the unique list. Will this cause any mutation? Therefore, adding a word "potentially" would be better.